splice/models.py

```diff
@@ -95,16 +97,18 @@ class Adgroup(db.Model):
     id = db.Column(db.Integer(), autoincrement=True, primary_key=True, info={"identity": [1, 1]})
     locale = db.Column(db.String(14), nullable=False)
 
-    type = db.Column(db.String(16))
-    paused = db.Column('paused', db.Boolean(), nullable=False, server_default=db.text(u'false'))
```

Comment:
We have existing data where Adgroups.type == NULL.  If this is necessary we need a strategy